pacai/agents/ghost/directional.py

```diff
@@ -47,7 +48,10 @@ def getDistribution(self, state):
             dist[a] = float(bestProb) / len(bestActions)
 
         for a in legalActions:
-            dist[a] += float(1 - bestProb) / len(legalActions)
+            try:
+                dist[a] += float(1 - bestProb) / len(legalActions)
+            except KeyError:
+                dist[a] = float(1 - bestProb) / len(legalActions)
```

Comment:
Instead of a try/except, let's test for the key and insert the default/base value (0.0 in this case).
I think this will be a bit more clear to future readers of this code.

Something like:
```
if (a not in dist):
    dist[a] = ...
dist[a] += ...
```